cfme/tests/cloud_infra_common/test_events.py

```diff
@@ -73,7 +73,11 @@ def test_vm_create(request, appliance, vm_crud, provider, register_event):
     request.addfinalizer(policy.delete)
 
     policy.assign_events("VM Create Complete")
-    request.addfinalizer(policy.assign_events)
+
+    @request.addfinalizer
+    def _cleanup():
+        policy.unassign_events("VM Create Complete")
```

Comment:
Given the repeated string use, it would be good to have an enumeration module for these event strings. That way, should they have to be version picked, we're only doing it in one place.

Eliminates chances of typos as well.

Not for this PR, but a later enhancement to consider.

Reply:
Definitely agree here, something to implement in the future.